exampleCourse/questions/demo/autograder/codeUpload-R-ipynb/info.json

```
@@ -0,0 +1,14 @@
+{
+  "uuid": "7ee46cad-fc03-4d1e-948a-0ef7b16688fb",
+  "title": "Demo: R, .ipynb file upload, external grading",
```

Nit, for consistency with other examples:

```suggestion
  "title": "Autograder demo: Upload R notebook",
```